lib/telnyx/calls.rb

```diff
@@ -0,0 +1,21 @@
+# frozen_string_literal: true
+
+module Telnyx
+  class Calls < APIResource
+    extend Telnyx::APIOperations::Create
+    extend Telnyx::APIOperations::NestedResource
+
+    ACTIONS = %w[reject answer hangup bridge speak fork_start fork_stop 
+    gather_using_audio gather_using_speak playback_start 
+    playback_stop record_start record_stop send_dtmf transfer]
+    
```

Comment:
I'm surprised rubocop didn't want to modify the indentation above?

Reply:
The old rubocop config seems to have removed, I didn't want to rerun it with the default config and change everything.